Fix way to obtain WAZUH_PATH

Hello team,

We have detected an issue with `deps/wazuh_testing/wazuh_testing/tools/__init__.py` in `3.13` and `3.14` introduced in this PR: https://github.com/wazuh/wazuh-qa/pull/700

That PR introduced the following change in the way `WAZUH_PATH` is obtained:
```
    with open("/etc/ossec-init.conf") as ossec_init:
        WAZUH_PATH = os.path.join(
            [item.rstrip().replace("DIRECTORY=", "").replace("\"", "")
            for item in ossec_init.readlines() if "DIRECTORY" in item][0])
```
In order to run our System tests we deploy some machines using Ansible and then launch the tests itself in our local host. The test will connect to those ansible machines and will perform appropriate actions. As the test itself will be running in our local machine it will fail because `/etc/ossec-init.conf` won't exists. Here is the error message:

```
../../tests/system/test_cluster/test_agent_enrollment/test_agent_enrollment.py:9: in <module>
    from wazuh_testing.tools.system import HostManager
wazuh_testing/tools/__init__.py:20: in <module>
    with open("/etc/ossec-init.conf") as ossec_init:
E   FileNotFoundError: [Errno 2] No such file or directory: '/etc/ossec-init.conf'
```
This error doesn't happen with `master` branch as this change was not applied yet.

The way `WAZUH_PATH` is obtained needs to be modified so the tests can run in environments without a wazuh installation.

--- a/deps/wazuh_testing/wazuh_testing/tools/__init__.py
+++ b/deps/wazuh_testing/wazuh_testing/tools/__init__.py
@@ -17,10 +17,13 @@ if sys.platform == 'win32':
     API_LOG_FILE_PATH = None
 
 else:
-    with open("/etc/ossec-init.conf") as ossec_init:
-        WAZUH_PATH = os.path.join(
-            [item.rstrip().replace("DIRECTORY=", "").replace("\"", "")
-            for item in ossec_init.readlines() if "DIRECTORY" in item][0])
+    if os.path.isfile("/etc/ossec-init.conf"):
+        with open("/etc/ossec-init.conf") as ossec_init:
+            WAZUH_PATH = os.path.join(
+                [item.rstrip().replace("DIRECTORY=", "").replace("\"", "")
+                for item in ossec_init.readlines() if "DIRECTORY" in item][0])
+    else:
+        WAZUH_PATH = os.path.join("/", "var", "ossec")
     WAZUH_CONF = os.path.join(WAZUH_PATH, 'etc', 'ossec.conf')
     WAZUH_API_CONF = os.path.join(WAZUH_PATH, 'api', 'configuration', 'api.yaml')
     WAZUH_SECURITY_CONF = os.path.join(WAZUH_PATH, 'api', 'configuration', 'security', 'security.yaml')

